lib/src/HttpUtils.cc

```diff
@@ -607,11 +607,15 @@ FileType parseFileType(const string_view &fileExtension)
 {
     // https://en.wikipedia.org/wiki/List_of_file_formats
     static const std::unordered_map<string_view, FileType> map_{
+        {"", FT_UNKNOWN},
         {"html", FT_DOCUMENT},
         {"docx", FT_DOCUMENT},
         {"zip", FT_ARCHIVE},
         {"rar", FT_ARCHIVE},
         {"xz", FT_ARCHIVE},
+        {"7z", FT_ARCHIVE},
+        {"tgz", FT_ARCHIVE},
+        {"gz", FT_ARCHIVE},
```

Comment:
I think we should add extensions with more than one dot, for example .tar.gz, do you think it would be a good idea to implement this?